packages/docusaurus/src/webpack/utils.ts

```diff
@@ -476,74 +483,7 @@ function getTerserParallel() {
   return terserParallel;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function getMinimizer(useSimpleCssMinifier = false): Plugin[] {
-  const minimizer = [
-    new TerserPlugin({
-      cache: true,
-      parallel: getTerserParallel(),
-      sourceMap: false,
-      terserOptions: {
-        parse: {
-          // we want uglify-js to parse ecma 8 code. However, we don't want it
-          // to apply any minification steps that turns valid ecma 5 code
-          // into invalid ecma 5 code. This is why the 'compress' and 'output'
-          // sections only apply transformations that are ecma 5 safe
-          // https://github.com/facebook/create-react-app/pull/4234
-          ecma: 8,
-        },
-        compress: {
-          ecma: 5,
-          warnings: false,
-        },
-        mangle: {
-          safari10: true,
-        },
-        output: {
-          ecma: 5,
-          comments: false,
-          // Turned on because emoji and regex is not minified properly using default
-          // https://github.com/facebook/create-react-app/issues/2488
-          ascii_only: true,
-        },
-      },
-    }),
-  ];
-
-  if (useSimpleCssMinifier) {
-    minimizer.push(
-      new OptimizeCSSAssetsPlugin({
-        cssProcessorPluginOptions: {
-          preset: 'default',
-        },
-      }),
-    );
-  } else {
-    minimizer.push(
-      ...[
-        new OptimizeCSSAssetsPlugin({
-          cssProcessorPluginOptions: {
-            preset: CssNanoPreset,
-          },
-        }),
-        new OptimizeCSSAssetsPlugin({
-          cssProcessor: CleanCss,
-          cssProcessorOptions: {
-            inline: false,
-            level: {
-              1: {
-                all: false,
-              },
-              2: {
-                all: true,
-                restructureRules: true,
-                removeUnusedAtRules: false,
-              },
-            },
-          },
-        }),
-      ],
-    );
-  }
-
-  return minimizer;
+  return [new ESBuildMinifyPlugin({target: 'es2017', css: true})];
```

Comment:
Wonder how much different this minification is compared to the old one?

Reply:
Earlier stat because I mess up the build shows that it reduces the output by 1%.